src/page-modules/contact/means-of-transport/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useMachine } from '@xstate/react';
+import { meansOfTransportFormMachine } from './means-of-transport-form-machine';
+import ModeOfTransportFormSelector from './means-of-transport-form-selector';
+import {
+  DelayForm,
+  DriverForm,
+  InjuryForm,
+  ServiceOfferingForm,
+  StopForm,
+  TransportationForm,
+} from './forms';
+
+export const MeansOfTransportContent = () => {
+  const [state, send] = useMachine(meansOfTransportFormMachine);
+
+  return (
+    <div>
```

Comment:
I think you should use <form> here and move the submit button to this component instead of having duplicate ones for each "form component".

Reply:
This way the "form components" only job is to decide which inputs to show